src/twisted/mail/pop3.py

```diff
@@ -855,7 +858,13 @@ def do_PASS(self, password):
 
         @type password: L{bytes}
         @param password: A password.
+
+        @type args: L{tuple} or L{None}
+        @param args: Other words composing the password.
```

Comment:
My bad, I copied the description from `do_AUTH` where that of `processCommand` would have been much better. Will change to:
```
@type args: L{tuple} of L{bytes}
@param args: Other words composing the password to be joined with a space.
```